WcaOnRails/spec/mailers/new_registration_mailer_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe NewRegistrationMailer, type: :mailer do
+  describe "send registration mail to new users" do
+    let(:user) { FactoryBot.create :user }
+    let(:mail) { NewRegistrationMailer.send_registration_mail(user) }
+
+    it "renders the headers" do
+      expect(mail.subject).to eq "Confirmation instructions"
+      expect(mail.to).to eq [user.email]
+      expect(mail.from).to eq ["notifications@worldcubeassociation.org"]
+    end
+
+    it "renders the body" do
+      expect(mail.body.encoded).to match user.email
+      expect(mail.body.encoded).to match "Your account is created and must be activated before you can use it."
+    end
+  end
+
+  describe "send registration mail to new users in their language" do
```

Comment:
I don't think it's terribly important to test this. As you already mentioned, these tests won't actually pass until the French translations are done anyways.

This sort of test is from the early days of internationalization on the WCA website when we were still trying to figure out how to do it right (that `localized_mail` method you're using is what we ended up building to make this easier on us).

Reply:
Alright, thanks for letting me know! I just hoped there is some way to check if either a translation is available or use the English text instead. But as it's not, I'll remove the translation test.